src/components/govukComponents/Label.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+import ReactHtmlParser from 'react-html-parser'
+
+class Label extends Component {
+  render() {
+
+    var attributes
+    if (this.props.attributes) {
+      attributes = this.props.attributes
+    }
+
+    if (this.props.html || this.props.text) {
+      const html = this.props.html
+      var label = <label className={`govuk-label${this.props.classes ? ' ' + this.props.classes : ''}`} {...attributes} htmlFor={this.props.for}>
+        {html ? ReactHtmlParser(html) : this.props.text}
+      </label>
+    }
+
+    if (this.props.isPageHeading === true) {
+      var pageheading = <h1 className="govuk-label-wrapper">{label}</h1>
+    }
+
+    return (
+      <>{pageheading ? pageheading : label}</>
```

Comment:
Is this fragment needed? Does it error without that?

Reply:
Yes, it doesn't like not having the fragment around this code
<img width="439" alt="Screen Shot 2019-04-08 at 11 16 39" src="https://user-images.githubusercontent.com/41327919/55716885-fdb40500-59ef-11e9-89d8-9a6968ab3cfb.png">